From e7b46ab023742c4dc6cb0599e5f9de6a7711125c Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Wed, 19 Feb 2014 19:16:11 +0800
Subject: [PATCH 0280/1074] ENGR00299749 ARM: dts: imx: add pxp v4l2 output
 support on imx6sx 17x17 arm2 board

Add V4L2 output via PxP support

Signed-off-by: Robby Cai <R63905@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 arch/arm/boot/dts/imx6sx-17x17-arm2.dts |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-17x17-arm2.dts b/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
index ea34d7f..f6af9ad 100644
--- a/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
+++ b/arch/arm/boot/dts/imx6sx-17x17-arm2.dts
@@ -21,6 +21,11 @@
 		default-brightness-level = <6>;
 	};
 
+	pxp_v4l2_out {
+		compatible = "fsl,imx6sx-pxp-v4l2", "fsl,imx6sl-pxp-v4l2";
+		status = "okay";
+	};
+
 	memory {
 		reg = <0x80000000 0x40000000>;
 	};
-- 
1.7.5.4

